Magento_Downloadable/templates/catalog/product/type.phtml

```diff
@@ -0,0 +1,26 @@
+<?php
+// @codingStandardsIgnoreFile
+?>
+
+<?php /* @var $block \Magento\Downloadable\Block\Catalog\Product\View\Type */ ?>
+<?php $_product = $block->getProduct() ?>
+
+<?php if ($block->displayProductStockStatus()): ?>
+    <?php if ($_product->getIsSalable()): ?>
+        <div
+            class="product-view__stock-status"
+            title="<?= __('Availability') ?>"
+        >
+            <?= __('In stock') ?>
+        </div>
+    <?php else: ?>
+        <div
+            class="product-view__stock-status product-view__stock-status--alert"
+            title="<?= __('Availability') ?>"
```

Comment:
Missing closing `>`

Reply:
Done

Reply:
Wrong place, text should be inside div ;)

Reply:
My fault 👍